### PR TITLE
Add AWS Console extension

### PR DESCRIPTION
**Pull request by Hodglim**
## Description

Add AWS Console extension.  The extension lets you easily launch AWS console links directly from Raycast.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/893162/144675374-82e010f7-ff4d-4b6b-b949-8a627fb77519.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @Hodglim 👋 

A smart way to open AWS services is not a bad idea, could it be an idea to somehow merge it into #316 (https://www.raycast.com/Falcon/aws) to have one extension — what do you think?

**Comment by Hodglim** [comment]
Hey @pernielsentikaer !
Definitely not against the idea and it would certainly be less confusing if there was just the one AWS extension in the store.
I will see what I can do.

**Comment by pernielsentikaer** [comment]
What do you think @victor-falcon - sounds like fine idea?

**Comment by victor-falcon** [comment]
Yes, of course! 👌

**Comment by Hodglim** [comment]
Hi @pernielsentikaer, I have now merged my extension functionality with @victor-falcon's extension.
Please can you both check and make sure you are happy and this is what you had in mind?
Thanks!

**Comment by victor-falcon** [comment]
In my opinion the subtitle of all commands should be "Amazon AWS" so that you can see which App is behind the command. What do you think @pernielsentikaer? 😃

**Comment by Hodglim** [comment]
You are correct @victor-falcon, that does seem to be the standard from looking at a few other extensions. 🤔

Personally I find having a whole list of commands with identical subtitles not particularly useful and prefer ​a bit more info on what each command does however to keep in line with other extensions I think you are right.

I know each command does also have a `description` property in package.json, however I don't think this is displayed in the UI?

File: extensions/amazon-aws/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "aws",
   "title": "Amazon AWS",
-  "description": "Integrate your AWS account in Raycast",
+  "description": "Integrate your AWS account in Raycast, by Falcon and Hodglim",
   "icon": "command-icon.png",
   "author": "Falcon",
```
Comment (pernielsentikaer) [inline extensions/amazon-aws/package.json:7]:
```suggestion
  "author": "Falcon",
  "contributors": [
    "Hodglim"
  ],
```

We're working on showing contributors, let's add it already 😄

File: extensions/amazon-aws/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "aws",
   "title": "Amazon AWS",
-  "description": "Integrate your AWS account in Raycast",
+  "description": "Integrate your AWS account in Raycast, by Falcon and Hodglim",
```
Comment (pernielsentikaer) [inline extensions/amazon-aws/package.json:5]:
```suggestion
  "description": "Integrate your AWS account in Raycast",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @Hodglim 👋 

Much better to have it in the current extensions 🔥  I left a few changes, could you look into them?

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

I added some feedback

File: extensions/amazon-aws/package.json (hunk 1 above)
Comment (Hodglim) [reply extensions/amazon-aws/package.json:7]: Ah perfect, this has now been updated as suggested 👍

**Review by mattisssa — APPROVED** [review]

**Pull request merged**